ast_data.c

```diff
@@ -96,6 +99,8 @@ const zend_ast_kind ast_kinds[] = {
 	ZEND_AST_FOREACH,
 };
 
+const size_t ast_kinds_count = sizeof(ast_kinds) / sizeof(ast_kinds[0]);
```

Comment:
This file is generated using `scripts/generate_ast_data.php`. The new node should be added in the there and then this file regenerated. (Can also rerun the stub script while at it.)